sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java

```diff
@@ -95,13 +98,25 @@ public static Builder newBuilder(SpanExporter spanExporter) {
     private long scheduleDelayMillis = SCHEDULE_DELAY_MILLIS;
     private int maxQueueSize = MAX_QUEUE_SIZE;
     private int maxExportBatchSize = MAX_EXPORT_BATCH_SIZE;
+    private boolean sampled;
```

Comment:
Please make this the default "true" :)

Reply:
Could you please explain why? This can be easily forgotten to configure and the spans with recorded events true will not be sent to the backend.

Reply:
1. This is the default for OpenCensus which has this functionality.
2. Only record one Span is usually used for in-memory debugging tools like z-pages in OpenCensus.

Reply:
I have made the change to report only sampled spans by default. Although It still seems confusing to me. 

Cross-linking the discussion with spec https://github.com/open-telemetry/opentelemetry-specification/issues/223#issuecomment-526247900